web_src/less/helpers.less

```diff
@@ -172,6 +172,8 @@
 .py-4 { padding-top: 1rem !important; padding-bottom: 1rem !important; }
 .py-5 { padding-top: 2rem !important; padding-bottom: 2rem !important; }
 
+.content-center { align-content: center !important; }
```

Comment:
Why important?

Reply:
Because all the other classes are important, I have written it as important as well.

Reply:
Important is ok for helpers.

Reply:
It's probably not necessary for this helper, but for others it definitely is to allow outweighing the heavyweight fomantic styles as well as the overly-nested styles of our own.

Reply:
Which simply means that we should fix our nested selectors and replace them with simpler ones.
But that's a topic for another PR.